core/src/main/java/org/eclipse/hono/util/TrustedCertificateAuthority.java

```diff
@@ -0,0 +1,143 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.util;
+
+import java.time.Instant;
+
+import org.eclipse.hono.annotation.HonoTimestamp;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+/**
+ * A trusted CA configuration.
+ *
+ */
+@JsonInclude(value = Include.NON_NULL)
+public class TrustedCertificateAuthority {
```

Comment:
IMHO we should declare this class final as I do not think that it is meant to be extended.